src/components/Lineage/Graph/index.tsx

```diff
@@ -94,63 +108,67 @@ export function EgeriaLineageGraph(props: Props) {
-                          actions={ <EgeriaLineageGraphActions rawData={{...rawData}} printUri={printUri}/> }
-                          onNodeClick={(d: any) => { setSelectedNodeData(d); setOpened(true); }} /> }
+          <Tabs.Panel value={LINEAGE_TYPES.END_TO_END}>
+            {!loading && (rawData.nodes.length === 0) && <EgeriaCantDisplay />}
+            {!loading && (rawData.nodes.length > 0) && <HappiGraph rawData={{ ...rawData }}
+              algorithm={'VISJS'}
+              debug={false}
+              graphDirection={'HORIZONTAL'}
+              selectedNodeId={guid}
+              actions={<EgeriaLineageGraphActions rawData={{ ...rawData }} printUri={printUri} />}
+              onNodeClick={(d: any) => { setSelectedNodeData(d); setOpened(true); }} />}
           </Tabs.Panel>
 
-          <Tabs.Panel value={ LINEAGE_TYPES.VERTICAL_LINEAGE }>
-            { !loading && (rawData.nodes.length === 0) && <EgeriaCantDisplay /> }
-            { !loading && (rawData.nodes.length > 0) && <HappiGraph rawData={{...rawData}}
-                            algorithm={'ELK'}
-                            debug={false}
-                            graphDirection={'VERTICAL'}
-                            selectedNodeId={guid}
-                            actions={ <EgeriaLineageGraphActions rawData={{...rawData}} printUri={printUri}/> }
-                            onNodeClick={(d: any) => { setSelectedNodeData(d); setOpened(true); }} /> }
+          <Tabs.Panel value={LINEAGE_TYPES.VERTICAL_LINEAGE}>
+            {!loading && (rawData.nodes.length === 0) && <EgeriaCantDisplay />}
+            {!loading && (rawData.nodes.length > 0) && <HappiGraph rawData={{ ...rawData }}
+              algorithm={'ELK'}
+              debug={false}
+              graphDirection={'VERTICAL'}
+              selectedNodeId={guid}
+              actions={<EgeriaLineageGraphActions rawData={{ ...rawData }} printUri={printUri} />}
+              onNodeClick={(d: any) => { setSelectedNodeData(d); setOpened(true); }} />}
           </Tabs.Panel>
 
-          <Tabs.Panel value={ LINEAGE_TYPES.ULTIMATE_SOURCE }>
-            { !loading && (rawData.nodes.length === 0) && <EgeriaCantDisplay /> }
-            { !loading && (rawData.nodes.length > 0) && <HappiGraph rawData={{...rawData}}
-                            algorithm={'VISJS'}
-                            debug={false}
-                            graphDirection={'HORIZONTAL'}
-                            selectedNodeId={guid}
-                            actions={ <EgeriaLineageGraphActions rawData={{...rawData}} printUri={printUri}/> }
-                            onNodeClick={(d: any) => { setSelectedNodeData(d); setOpened(true); }} /> }
+          <Tabs.Panel value={LINEAGE_TYPES.ULTIMATE_SOURCE}>
+            {!loading && (rawData.nodes.length === 0) && <EgeriaCantDisplay />}
+            {!loading && (rawData.nodes.length > 0) && <HappiGraph rawData={{ ...rawData }}
```

Comment:
not consistent with the whitespaces here, in some places you removed them, in some other place you added, please decide

Reply:
It was done by the auto formatting, I've seen that you used rawData={{...rawData }} formatting while it reccomends rawData={{ ...rawData }} instead. I'll stick with yours, looks better

Reply:
fixed